### DISCUSSION TITLE
Transferring Settings to PrusaSlicer

### DISCUSSION THREAD

**Discussion by VrtlBrown** [supermerill/SuperSlicer#3628] [Q&A] [answered]
Love the community here and SuperSlicer in general. But in this pause in development I want to nip over to PrusaSlicer to use the new features there.

Is there a way to transfer all my settings over? I tried the obvious export/import options but they didn’t seem to work.

**Comment by neophyl** [comment, marked as answer]
Simple answer is no.

**Reply by Bumblebiber** [reply]
Go to the settings folder, copy your settings (print, printer, filament) into the prusaslicer settings folder. 

It might give you some errors, as there are parameters in SS that PS doesn't know. But look closely at the error message, note the parameter name, open the .ini file with an editor and simply remove that line.

**Reply by neophyl** [reply]
and if you make one error you can ruin the profile.  You might as well just open up both side by side and copy the settings for a profile manually from one to the other using the interface.  Safer and will take about as long.  
Its not like there are that many PS settings anyway compared to SuSi.  

There is no programmed way.  Unless you have written a conversion script that can parse a SuSI config bundle export and remove the ones PS doesn't understand, as well as changing the values for the settings that are common but are treated differently by the 2 programs ?  I'm sure people would be interested if you have written something like that.
There are settings though that while they are in both configurations are actually different.  Like elephants foot, a positive value in PS moves perimeters inwards (shrinks them) while in SuSi those are negative for inwards and positive to grow the perimeters.  

So just going by any 'errors' does not get you a profile that works the same.

**Comment by VrtlBrown** [comment]
Thanks for the replies. Yeah in the end I just opened them both side by side and copied each relevant setting, wasn’t too bad in the end

**Comment by chrisdixonstudios** [comment]
to bring Prusa settings to SuperSlicer I open a Prusa .3mf project ..then side by side compare goes much faster. Should work the other way into Prusaslicer.

**Comment by julian-weinert** [comment]
I'm trying my best to get away freom SuperSlicer as well, but apparently a floating point number is an invalid value foir extrusion width in PrusaSlicer. So I guess I'm still stuck with this non functioning SuperSlicer stuff and if I happen to design a model that crashes this barely working program again I'll simply forget about it and cancel the project
